### ISSUE TITLE
Hyperparameters for loss functions

### ISSUE THREAD

**Issue by liboyu02** [apple/ml-depth-pro#94]
Thanks for your work! Could you please provide your hyperparameters for your loss functions?
